src/main/java/com/bunq/sdk/context/ApiContext.java

```diff
@@ -124,14 +123,20 @@ public static ApiContext restore(String fileName) {
     try {
       File file = new File(fileName);
       String json = FileUtils.readFileToString(file, ENCODING_BUNQ_CONF);
-      JsonObject jsonObject = gson.fromJson(json, JsonObject.class);
 
-      return gson.fromJson(jsonObject, ApiContext.class);
+      return fromJson(json);
     } catch (IOException exception) {
       throw new BunqException(ERROR_COULD_NOT_RESTORE_API_CONTEXT, exception);
     }
   }
 
+  /**
+   * Restores a context from a given JSON string.
+   */
+  public static ApiContext fromJson(String json) {
+    return gson.fromJson(json, ApiContext.class);
+  }
+
   private void initialize(String deviceDescription, List<String> permittedIps) {
     /* The calls below are order-sensitive: to initialize a Device Registration, we need an
      * Installation, and to initialize a Session we need a Device Registration. */
```

Comment:
Ah `The calls below`... 🤔 explains why its below the method 😁 

But then the formatting ? Shouldn't it be as 👆